guide/filters.rst

```diff
@@ -0,0 +1,6 @@
+Filters
+=======
+
+The different types of assimilation algorithms (EAKF, ENKF, Kernel filter, Particle filter, etc.) are determined by the
+``&assim_tools_nml:filter_kind`` entry, described in :doc:`../../assimilation_code/modules/assimilation/assim_tools_mod`. Despite having
+'filter' in the name, they are assimilation algorithms and so are implemented in ``assim_tools_mod.f90``.
```

Comment:
This document should be removed because it has so little content. 
The navigation bar should just point to assim_tools_mod.rst